tasking/serializers/contenttype.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+ContentType serializer
+"""
+from __future__ import unicode_literals
+
+from django.contrib.contenttypes.models import ContentType
+
+from rest_framework import serializers
+from rest_framework.validators import UniqueTogetherValidator
+
+
+# pylint: disable=abstract-method
```

Comment:
what caused this?

Reply:
This was just leftover / forgotten code. Removed in latest commit.